channel/connection_caretaker.h

```diff
@@ -77,13 +74,9 @@ class ConnectionCaretaker {
   // Sends one heartbeat. It is public so that timer callback can use it.
   void send_heartbeat();
 
-  // Forces a synchronous refresh of the authz token if it's expired or due to
-  // expire within the notice period.
-  void refresh_authz_token();
-
 private:
   // Sends following information:
-  //   agent verstion, api_key (including tenant) and any config labels.
```

Comment:
Maybe change the "verstion" typo while you're at it?

Reply:
Done.